e2e_tests/tests/nightly/test_distributed.py

```diff
@@ -284,6 +284,7 @@ def test_remote_search_runner() -> None:
 
 @pytest.mark.distributed
 @pytest.mark.gpu_required
+@pytest.mark.huggingface
```

Comment:
I think just erroring out when the env var is missing with a detailed message is best

Reply:
For dev experience, it would make sense for there to be a `skipif` clause for these tests, but I'm not sure that we should be mixing `@pytest.mark.distributed` with `@pytest.mark.huggingface` in the first place. This should probably be its own separate workflow. 

Distributed tests are meant to test just that--distributed workloads. Testing that we can download weights from huggingface should be its own separate thing.

1) Can we verify that weights download properly in a separate, huggingface-specific workflow?
2) Can this example be tested with random weights? Can we bypass the weight download step by having it already in our file system?

Reply:
> but I'm not sure that we should be mixing @pytest.mark.distributed with @pytest.mark.huggingface in the first place

I probably misunderstood, but I just tried to follow what I thought was done for [the deepspeed tests](https://github.com/determined-ai/determined/blob/6d4e9f039fe6e2c54dd202edb0598aa4d5b48a52/e2e_tests/tests/nightly/test_distributed.py#L385-L387) where multiple marks were applied. Not the right thing to do?

Reply:
>Testing that we can download weights from huggingface should be its own separate thing.

> Can we verify that weights download properly in a separate, huggingface-specific workflow?
Can this example be tested with random weights? Can we bypass the weight download step by having it already in our file system?

Good points, will look into it.

Reply:
> Can this example be tested with random weights?
Not easily. Would require a large re-write of the example.

> Can we bypass the weight download step by having it already in our file system?

Is this possible? Do you know if we already do this for any other examples @MikhailKardash